jekyll/_cci2/notifications.md

```diff
@@ -1,8 +1,31 @@
 ---
 layout: classic-docs
-title: "Notifications"
-short-title: "Notifications"
+title: "Enabling Web Notifications"
+short-title: "Enabling Web Notifications"
 categories: [configuring-jobs]
 order: 100
-published: false
+published: true
 ---
+
+## Enabling Web Notifications
+
+1. Go to your [CircleCI account settings](https://circleci.com/account). The link to turn on permissions is at the bottom in the Web Notifications section as shown in the screenshot:
```

Comment:
'account settings' is labelled 'User Settings' in both the current UI and Beta UI:

![image](https://user-images.githubusercontent.com/809823/27423721-77be9ab0-572a-11e7-9dcf-d71adcf5031c.png)

![image](https://user-images.githubusercontent.com/809823/27423741-881bed90-572a-11e7-8a21-86abe328be5e.png)